SUPPORT.md

```diff
@@ -0,0 +1,17 @@
+# Support
+
+## How to file issues and get help
+
+This project uses [GitHub issues][gh-issue] to [track bugs][gh-bug] and [feature requests][gh-feature]. Please search the existing issues before filing new issues to avoid duplicates. For new topics, file your bug or feature request as a new issue.
+
+For questions about using this project, please look at the [Demo repo for CBL-Mariner][demo] and our [Contributor's Guide][contributor] if you want to work on CBL-Mariner.
+
+## Microsoft Support Policy
+
+Support for CBL-Mariner is limited to the resources listed above.
+
+[gh-issue]: https://github.com/microsoft/CBL-Mariner/issues/new/choose
+[gh-bug]: https://github.com/microsoft/CBL-Mariner/issues/new?assignees=&labels=Issue-Bug&template=bug_report.md&title=
```

Comment:
It looks to me like the `gh-feature` and `gh-bug` links reference a couple of bug templates that I can't actually find in our `.github` folder. Additionally, when I go to those URLs, it doesn't fill out the labels like the query strings would imply.